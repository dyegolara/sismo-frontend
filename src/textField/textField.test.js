/* eslint-env jest */
import React from 'react'
import renderer from 'react-test-renderer'

import TextField from './index'

describe('<TextField />', () => {
  test('matches snapshot with default props', () => {
    let $textField = renderer.create(<TextField />)
    expect($textField).toMatchSnapshot()
  })
})
