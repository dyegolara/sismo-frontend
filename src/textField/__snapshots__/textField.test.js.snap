// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextField /> matches snapshot with default props 1`] = `
<div
  className="control"
>
  <p>
    <input
      className="input "
      disabled={false}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      placeholder=""
      required={false}
      type="text"
      value=""
    />
  </p>
  <span
    className="help is-danger is-hidden"
  >
    
  </span>
</div>
`;
